recipes-core/uci/uci_git.bbappend

```diff
@@ -0,0 +1,72 @@
+################################################################################
+#
+# uci_git.bb
```

Comment:
Should be uci_git.bbappend ?
Please check other bbappends as well

Reply:
fixed